lib/model/ui/user_ui_model.dart

```diff
@@ -0,0 +1,34 @@
+class UserUiModel {
+  final String avatarUrl;
+  final String name;
+  final String description;
+
+  UserUiModel({
+    required this.avatarUrl,
+    required this.name,
+    required this.description,
+  });
+
+  static List<UserUiModel> fakeData() {
+    return [
+      UserUiModel(
```

Comment:
What do you think about using a loop here?
It's a good example when creating fake data, cause I was faced with an issue in the varun project that we have to keep repeat updates when we add/remove/change a field in an `UiModel` 😢